hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/DiskBalancerConfiguration.java

```diff
@@ -36,6 +38,17 @@ public final class DiskBalancerConfiguration {
   private static final Logger LOG =
       LoggerFactory.getLogger(DiskBalancerConfiguration.class);
 
+  // The path where datanode diskBalancer's conf is to be written to.
+  public static final String HDDS_DATANODE_DISK_BALANCER_INFO_DIR =
```

Comment:
These configuration properties should be put to ozone-default.xml

Reply:
@lokeshj1703 Could you help to check this comment?

Reply:
Please add comments that if this property is not defined, ozone will fallback to use meta data directory instead.